app/assets/stylesheets/shared/helpers/_player-container.scss

```diff
@@ -42,7 +42,7 @@
     button,
     a {
       &:focus {
-        background-color: orange;
+        background-color: #ffa500;
```

Comment:
Non-blocker: we can replace this with `$govuk-focus-colour` (in case the Design System decides to review the colours in future) – we should have access to this `govuk-frontend` variable through `govuk_publishing_components`.

Reply:
Thanks for the tip.

Unfortunately govuk-frontend is available in some of the stylesheets but not the admin ones so this doesn't work in all contexts 😞